frontend/src/components/reviews-replies/ReviewForm.vue

```diff
@@ -8,22 +8,32 @@
               to inform others -- other students, teachers and the university.
               Please think carefully about your feedback before you submit it.
             </p>
-            <AppInput placeholder="Review title..." v-model="title"/><br>
-            <textarea placeholder="Your review here.." v-model="body"></textarea><br>
+            <span class="required">*</span>
+            <AppInput placeholder="Review title..." v-model="title" style="margin:8px auto;"/><br>
+            <span class="required">*</span>
+            <textarea placeholder="Your review here..." v-model="body"></textarea><br>
 
-            <InputOptions v-model="recommend" :options="['Yes', 'No', 'Unsure']">
-              Would you recommend this course?
+            <InputOptions v-model="recommend" :options="['Yes', 'No']">
+              Would you recommend this course to a friend? <span class="required">*</span>
             </InputOptions>
 
-            <InputOptions v-model="enjoy" :options="['Yes', 'No', 'Unsure']">
-              Did you enjoy this course?
+            <InputOptions v-model="enjoy" :options="['1', '2', '3', '4', '5']">
```

Comment:
Still unsure on numbers. I think simplified choices are generally best.

Reply:
I'm not 100% sure either...did this on a whim.
But I actually think mixing up the method of input is worthwhile.
I think we should discuss tomorrow and figure out exactly what to put on reviews for now.